File: day1.py
with open("day1.txt","r") as f:
    lines = f.read().splitlines()

count = 0
for i in range(1, len(lines)):
    if int(lines[i]) > int(lines[i-1]):
        count+=1
print('Part 1: ', count)

count = 0
for i in range(len(lines)-3):
    if int(lines[i+3]) > int(lines[i]):
        count+=1
print('Part 2: ', count)

File: day1.rs
mod utils;

fn main() {
    let lines = utils::read_file_as_num("./day1.txt");
    
    let mut count = 0;
    for i in 1..lines.len() {
        if lines[i] > lines[i-1] {
            count += 1;
        }
    }
    println!("Part 1: {}", count);

    count = 0;
    for i in 0..lines.len()-3 {
        if lines[i+3] > lines[i] {
            count += 1;
        }
    }
    println!("Part 2: {}", count);
}
